solidity/test/DepositRedemptionTest.js

```diff
@@ -674,6 +674,80 @@ describe("DepositRedemption", async function() {
     })
   })
 
+  describe("transferAndRequestRedemption", async () => {
+    const sighash =
+      "0xb68a6378ddb770a82ae4779a915f0a447da7d753630f8dd3b00be8638677dd90"
+    const outpoint = "0x" + "33".repeat(36)
+    const valueBytes = "0x1111111111111111"
+    const keepPubkeyX = "0x" + "33".repeat(32)
+    const keepPubkeyY = "0x" + "44".repeat(32)
+    // Override redeemer output script for this test.
+    // No real reason here, it's just how we derived the below values.
+    const redeemerOutputScript = "0x160014" + "33".repeat(20)
```

Comment:
I'm guessing this isn't a new vector… Where are we reusing it from, and how can we avoid duplicating the test vector?

Reply:
it is indeed not. It's from `requestRedemption` right above. Why do we want to avoid duplicating this test vector? It's a path that can be taken towards redemption and it should be run fully imo. It also ensures any future changes to this function don't interfere with the redemption intent, or at least catches intent

Reply:
For me, understanding all the test vectors we use is a way to understand how good our coverage is of various paths. Duplicate vectors testing different paths basically give us an opportunity to drift vectors away from each other without questioning why the divergence is happening. Having them centralized lets us inspect any decision to change a vector in the context of what other tests are using that vector, and understand why both tests can bear the change or why we might need a completely separate vector in a given case. 

Somewhat relatedly, it also lets the test be as descriptive as possible by making the code focus on test intent rather than setup. 

Of course all of this assumes the vectors are named reasonably well as well—encapsulating what the vector represents. I’ve tried to do that with our existing shared vectors in utils, though we still have a generically-named “TX” that “does not spark joy” heh.

Reply:
We never circled back here---I'm comfortable with merging this as is and doing a PR to fix the duplication, though.